_episodes/11.md

```diff
@@ -0,0 +1,244 @@
+inner loop into memory to avoid a lot of extra disk calls. Since it is ideal to
+read an entire table into memory, you will likely want to make sure the table
+that is built in memory is the smaller size of the two tables. This smaller
+table called the build table. The table that gets streamed is called the probe
+table. We discussed a bit how hash-joins work which is a common mechanism to
+execute joins in a distributed and parallel fashion.  
+
+Another nomenclature akin to build table and probe tables are dimension and
+fact table, respectively. This nomenclature comes from the [star schema](https://en.wikipedia.org/wiki/Star_schema)
+that comes from data warehousing. Typically, there are large tables called fact
+tables that would live at the center of the schema. These tables typically have
+many foreign keys, and a bit of quantitative or measuarable columns of the event
+or instance. The foreign keys connect these big fact tables to smaller dimension
+tables that when joined provide human readable context to enrich the recordings
+in the fact table. The schema ends up looking like a star with the fact table at
+the center. In essence, you just need to remember when someone is describing a
+fact table they are saying it is a bigger table that is likely going to end up
+on the probe side of a join, where a dimension is more likely a candidate to fit
+into memory on the build side of a join.
+
+So let's get onto the Dynamic Filtering shall we? First,let's cover a few
+concepts about Dynamic Filtering, then cover some variations of this concept.
+
+Dynamic Filtering takes advantage of joins with big fact tables to smaller
+dimension tables. What makes this filtering different from other types of
+filtering is that you are using the smaller build table that is loaded at query
+time to generate a list of values that exist in the join column between the
+build table and probe table. We know that only values that match these criteria
+are going to be returned from the sales side, so we can use this dynamically
+generated list as a pushdown predicate on the join column of the probe side.
+This means we are still scanning this data but only sending the subset that will
+answer the query. We can look at the blog written for the original local dynamic
+filtering implementation by Roman Zeyde for more insights on the original
+implementation for Dynamic Filtering before Raunaq's changes.
+
+<https://trino.io/blog/2019/06/30/dynamic-filtering.html>
+
+Local Dynamic Filtering is definitely beneficial, in that it generates a list
+of values that can be used to avoid sending some unnecessary rows, but what if 
+we could cut out entire splits of unnecessary partitions from the query
+execution? In the next iteration of Dynamic Filtering, called Dynamic Partition 
+Pruning, we can take advantage of probe tables that are partitioned and use that
+information to tell the coordinator to prune or eliminate the splits that do not
```

Comment:
... tell the coordinator to avoid generating splits from partitions that can be eliminated based on the dynamic filters.

Maybe split up this sentence